Riot/Modules/Room/Views/BubbleCells/Call/Direct/RoomDirectCallStatusBubbleCell.swift

```diff
@@ -0,0 +1,207 @@
+// 
+// Copyright 2020 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+
+class RoomDirectCallStatusBubbleCell: RoomBaseCallBubbleCell {
+    
+    private enum Constants {
+        static let statusTextFontSize: CGFloat = 14
+        static let statusTextInsets: UIEdgeInsets = UIEdgeInsets(top: 8, left: 8, bottom: 12, right: 8)
+        // swiftlint:disable force_unwrapping
+        static let statusCallBackURL: URL = URL(string: "element://call")!
```

Comment:
App URL scheme is now in `BuildSettings`

Reply:
This is not used as a url scheme actually, it's just used to define a url.